scrapeNews/scrapeNews/spiders/asianage.py

```diff
@@ -53,7 +53,7 @@ def getPageTitle(self, newsBox):
 
     def getPageLink(self, newsBox):
         data = 'http://www.asianage.com/newsmakers' + newsBox.xpath('div/a/@href').extract_first()
-        if (data is None):
+        if (data == 'http://www.asianage.com/newsmakers'):
             loggerError.error(newsBox)
```

Comment:
Improve error Logging! How are you supposed to know what data was missing!

Reply:
Update for logging is under development! One PR must contain only one logical change :)